spring-boot-project/spring-boot-docs/src/docs/asciidoc/spring-boot-features.adoc

```diff
@@ -6948,7 +6948,7 @@ You can use the `@DataJpaTest` annotation to test JPA applications.
 By default, it scans for `@Entity` classes and configures Spring Data JPA repositories.
 If an embedded database is available on the classpath, it configures one as well.
 SQL queries are logged by default by setting the `spring.jpa.show-sql` property to `true`.
-This can be disabled using the `showSql()` attribute of the annotation.
+This can be disabled by using the `showSql()` attribute of the annotation.
```

Comment:
I think this is OK without the "by".

Reply:
@wilkinsona I just felt that it was clearer, but it seems that my lack of knowledge on English made me feel that way. Thanks for pointing it out 😄